workspaces/adventure-pack/goodies/python3/src/is_palindrome/test.py

```diff
@@ -0,0 +1,22 @@
+from . import *
+
+
+def test_even_fail():
+    assert not is_palindrome("ABCBBA")
+
+
+def test_odd_fail():
+    assert not is_palindrome("1234221")
+
+
+def test_pass():
```

Comment:
I like that we have explicit even and odd fails, could we have the same for pass?